packages/storybook-test/src/WcagAuditReport.tsx

```diff
@@ -1,23 +1,105 @@
 import { useOf } from '@storybook/blocks';
 import uniq from 'lodash-es/uniq';
+import without from 'lodash-es/without';
+import { successCriteria, versionSort } from './wcag22';
 import { isWcagTest, WCAG22_AA_LEVEL, WcagAudit, WcagTest } from './WcagAudit';
 
+interface WcagReference {
+  href: string;
+  linkText: string;
+  sortKey: string;
+}
+
 export function WcagAuditReport() {
-  const resolvedOf = useOf<'story'>('story');
-  const wcagAudit: WcagAudit = resolvedOf.story.parameters['wcagAudit'];
+  const meta = useOf<'meta'>('meta');
+  const wcagAudit: WcagAudit = meta.csfFile?.meta?.parameters && meta.csfFile?.meta?.parameters['wcagAudit'];
+  const wcagStories = Object.values(meta.csfFile?.stories || {}); //.filter((story) => story.parameters.wcagAudit);
+
+  const baselineVersions = [
+    {
+      date: '2024-12-01',
+      href: 'https://nldesignsystem.nl/baseline/2024-12',
+      text: 'NL Design System Baseline: december 2024',
+    },
+    {
+      date: '2025-01-01',
```

Comment:
Waarom hebben we alvast een Baseline versie voor januari?

Reply:
oops, dat was een test